js/adapt-contrib-slider.js

```diff
@@ -383,26 +383,26 @@ define([
             this.$('.slider-markers').empty();
```

Comment:
While we're here could we convert this line to a variable declaration i.e.
```js
var $markers = this.$('.slider-markers').empty();
```
whilst replacing the occurence of `this.$('.slider-markers')` on line 385 and removing line 391?